File: src/main/java/no/fintlabs/dispatch/model/InstanceHeadersEntity.java
package no.fintlabs.dispatch.model;

import lombok.*;

import javax.persistence.Entity;
import javax.persistence.Id;

@Entity
@Getter
@Setter
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class InstanceHeadersEntity {
    @Id
    private String sourceApplicationInstanceId;
    private String sourceApplicationIntegrationId;
    private String archiveInstanceId;
}

File: src/main/kotlin/no/novari/flyt/egrunnerverv/gateway/dispatch/model/InstanceHeadersEntity.kt
package no.novari.flyt.egrunnerverv.gateway.dispatch.model

import jakarta.persistence.Entity
import jakarta.persistence.Id

@Entity
class InstanceHeadersEntity(
    @Id
    var sourceApplicationInstanceId: String = "",
    var sourceApplicationIntegrationId: String = "",
    var archiveInstanceId: String = "",
)
